src/modules/core/components/TokenEditDialog/TokenEditDialog.tsx

```diff
@@ -124,6 +139,27 @@ const TokenEditDialog = ({ addTokenFn, tokens = [], cancel, close }: Props) => {
           values,
         }: FormikProps<FormValues>) => (
           <>
+            <DialogSection appearance={{ theme: 'sidePadding' }}>
+              {allTokens.length > 0 ? (
```

Comment:
I mean there's always going to be at least 2 tokens:
- native colony token
- network token

So this check does not need to exist